__snapshots__/make-badge.spec.mjs.js

```diff
@@ -2551,3 +2482,1593 @@ exports['The badge generator "social" template badge generation should match sna
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  width="113"
+  height="20"
+  role="img"
+  aria-label="label: message"
+>
+  <title>label: message</title>
+  <linearGradient id="s" x2="0" y2="100%">
+    <stop offset="0" stop-color="#bbb" stop-opacity=".1" />
+    <stop offset="1" stop-opacity=".1" />
+  </linearGradient>
+  <clipPath id="r">
+    <rect width="113" height="20" rx="3" fill="#fff" />
+  </clipPath>
+  <g clip-path="url(#r)">
+    <rect width="54" height="20" fill="#555" />
+    <rect x="54" width="59" height="20" fill="#4c1" />
+    <rect width="113" height="20" fill="url(#s)" />
+  </g>
+  <g
+    fill="#fff"
+    text-anchor="middle"
+    font-family="Verdana,Geneva,DejaVu Sans,sans-serif"
+    text-rendering="geometricPrecision"
+    font-size="110"
+  >
+    <image
+      x="5"
+      y="3"
+      width="14"
+      height="14"
+      xlink:href="data:image/svg+xml;base64,PHN2ZyB4bWxu"
+    />
+    <text
+      aria-hidden="true"
+      x="365"
+      y="150"
+      fill="#010101"
+      fill-opacity=".3"
+      transform="scale(.1)"
+      textLength="270"
+    >
+      label
+    </text>
+    <text x="365" y="140" transform="scale(.1)" fill="#fff" textLength="270">
+      label
+    </text>
+    <text
+      aria-hidden="true"
+      x="825"
+      y="150"
+      fill="#010101"
+      fill-opacity=".3"
+      transform="scale(.1)"
+      textLength="490"
+    >
+      message
+    </text>
+    <text x="825" y="140" transform="scale(.1)" fill="#fff" textLength="490">
+      message
+    </text>
+  </g>
+</svg>
+
+`
+
+exports['The badge generator badges with logos should always produce the same badge default badge with logo, no label, has message 1'] = `
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  width="141"
+  height="20"
+  role="img"
+  aria-label="undefined: message"
+>
+  <title>undefined: message</title>
+  <linearGradient id="s" x2="0" y2="100%">
+    <stop offset="0" stop-color="#bbb" stop-opacity=".1" />
+    <stop offset="1" stop-opacity=".1" />
+  </linearGradient>
+  <clipPath id="r">
+    <rect width="141" height="20" rx="3" fill="#fff" />
+  </clipPath>
+  <g clip-path="url(#r)">
+    <rect width="82" height="20" fill="#555" />
+    <rect x="82" width="59" height="20" fill="#4c1" />
+    <rect width="141" height="20" fill="url(#s)" />
+  </g>
+  <g
+    fill="#fff"
+    text-anchor="middle"
+    font-family="Verdana,Geneva,DejaVu Sans,sans-serif"
+    text-rendering="geometricPrecision"
+    font-size="110"
+  >
+    <image
+      x="5"
+      y="3"
+      width="14"
+      height="14"
+      xlink:href="data:image/svg+xml;base64,PHN2ZyB4bWxu"
+    />
+    <text
+      aria-hidden="true"
+      x="505"
+      y="150"
+      fill="#010101"
+      fill-opacity=".3"
+      transform="scale(.1)"
+      textLength="550"
+    >
+      undefined
+    </text>
+    <text x="505" y="140" transform="scale(.1)" fill="#fff" textLength="550">
+      undefined
+    </text>
```

Comment:
At the layer of the application we are testing, to say this badge has an empty `label`, we need to pass `label: ''`. If you actually pull this image out and view it, the image you're generating here (and asserting against) is

![undefined](https://github.com/user-attachments/assets/b47e1a66-190c-4b87-9b4d-6d02ca571a23)

This is what comes out when you give the function that input, but it wasn't really what you're trying to test here.